configs/edison/job_script.edison.bash

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -l
+
+# comment out if using debug queue
+#SBATCH --partition=regular
+# comment in to get premium queue
+##SBATCH --qos=premium
+# comment in to get the debug queue
+##SBATCH --partition=debug
+# change number of nodes to change the number of parallel tasks
+#SBATCH --nodes=10
+#SBATCH --time=1:00:00
+#SBATCH --account=acme
+#SBATCH --job-name=mpas_analysis
+#SBATCH --output=mpas_analysis.o%j
+#SBATCH --error=mpas_analysis.e%j
+#SBATCH -L cscratch1,SCRATCH,project
+
+cd $SLURM_SUBMIT_DIR   # optional, since this is the default behavior
+
+export OMP_NUM_THREADS=1
+
+module unload python python/base
+module use /global/project/projectdirs/acme/software/modulefiles/all
+module load python/anaconda-2.7-acme
+
+# change this to the name of the run you would like to analyze
+run_name="20170313.beta1.A_WCYCL1850S.ne30_oECv3_ICG.edison"
+# by default, output to a scratch directory with the name of the run,
+# but change this as desired
+output_dir="$SCRATCH/mpas_analysis/$run_name"
```

Comment:
no no, I understand we want to leave the config filename. I was thinking mostly of `run_name` and `output_dir` only.